pmm/aws-staging-start.groovy

```diff
@@ -313,6 +317,30 @@ pipeline {
                 }
             }
         }
+        stage('Enable Experimental Repo') {
+            when {
+                expression { env.PMM_VERSION == "pmm2" && env.CLIENT_INSTANCE == "no" && env.ENABLE_RC_REPO == "yes" }
+            }
+            steps {
+                script {
+                    withEnv(['JENKINS_NODE_COOKIE=dontKillMe']) {
+                        sh """
+                            export IP=\$(cat IP)
+                            export VM_NAME=\$(cat VM_NAME)
+                        """
+                        node(env.VM_NAME){
+                            sh """
+                                set -o errexit
+                                set -o xtrace
+                                docker exec \${VM_NAME}-server sed -i'' -e 's^/release/^/testing/^' /etc/yum.repos.d/pmm2-server.repo
```

Comment:
Shouldn't we use experimental repo for server packages in this case?
@denisok ?

Reply:
we can do that too

Reply:
@EvgeniyPatlan changing pmm2-server RC repo to experimental may require changes also on release job and probably some other places, hence I have not tried to make that change, we can make this consistent before next release, for current release could we just go ahead with what we have done already? i.e use testing for RC of pmm2-server and experimental for pmm2-client.

Reply:
I could agree here, so far it is more like a PoC, to gather all requirements and to change it completely for the 2.17